components/centraldashboard/public/components/main-page.js

```diff
@@ -291,6 +291,18 @@ export class MainPage extends utilitiesMixin(PolymerElement) {
         // A temporary workaround is  to use "this.queryParams" as an input
         // instead of "queryParamsChange.base".
         // const queryParams = queryParamsChange.base;
+        const queryParams = this.queryParams;
+        if (queryParams) {
+            if (queryParams["ns"]) {
+                if (queryParams["ns"] !== null){
+                    return this.buildHref(
```

Comment:
Could you restructure the if statements in order to reduce the nesting levels?

Something like:
```javascript
if (!queryParams || !queryParams["ns"]) {
  return this.buildHref(href, this.queryParams);
}

return this.buildHref(href.replace('{ns}', queryParams["ns"]), queryParams);
```

Reply:
@kimwnasptd Good idea ! I corrected this part, please look at my latest PR.
I inserted a line break not to exceed 80 chars.

Reply:
LGTM!